packages/init/src/index.ts

```diff
@@ -80,7 +80,10 @@ async function init() {
     isTrackingAvailable
   );
 
-  Tracker.get().trackInitDone(frameworkResult.value, repositoryDomainName);
+  await Tracker.get().trackInitDone(
```

Comment:
I see how it would fix the issue of the Init event not being sent,
From a user perspective, does it run smoothly ?

What I means is, it is located between 2 steps, if we wait for it, does that means the user doesn't see any messages or anything happening until the event is sent ?

Reply:
means it takes a second or two longer before showing the message.

Reply:
I think for that part we might want to Ask designers and UX what they think you know.